sql/core/src/test/scala/org/apache/spark/sql/test/SharedSQLContext.scala

```diff
@@ -17,4 +17,17 @@
 
 package org.apache.spark.sql.test
 
-trait SharedSQLContext extends SQLTestUtils with SharedSparkSession
+trait SharedSQLContext extends SQLTestUtils with SharedSparkSession {
+
+  override protected val doThreadAuditInSparkFunSuite = false
+
+  protected override def beforeAll(): Unit = {
+    doThreadPreAudit
+    super.beforeAll
```

Comment:
nit: `super.beforeAll()`, and also `super.afterAll()`.

Reply:
Fixed.